gravitee-apim-console-webui/src/management/api/creation-v4/steps/step-3-endpoints/step-3-endpoints-1-list.component.ts

```diff
@@ -96,6 +97,18 @@ export class Step3Endpoints1ListComponent implements OnInit, OnDestroy {
       .subscribe();
   }
 
+  private mapAndFilterEndpoints(endpointPlugins: ConnectorPlugin[], requiredQoS: Qos[]) {
+    return endpointPlugins
+      .map((endpoint) => fromConnector(this.iconService, endpoint))
+      .filter((endpoint) => {
```

Comment:
I would apply the filter only if `requiredQoS.length > 0`

Reply:
Updated with this check.